src/main/kotlin/lotto/domain/LottoList.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.domain
+
+import lotto.domain.lottoStrategy.LottoStrategy
+
+data class LottoList(
+    val lottos: List<Lotto>
+) {
+    fun size(): Int = lottos.size
+
+    fun getResult(previousLottoNumbers: LottoNumbers): LottoResult =
+        LottoResult(
+            lottos
+                .groupBy { it.countMatchNumbers(previousLottoNumbers) }
+                .mapValues { LottoList(it.value) }
+        )
```

Comment:
아래 처럼도 작성할 수 있어요!
참고만 하셔도 괜찮습니다 :)
```suggestion
            return lottos
                .groupBy { it.countMatchNumbers(previousLottoNumbers) }
                .mapValues { LottoList(it.value) }
                .let { LottoResult(it) }
```